lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -20,7 +20,7 @@ blockchain_configuration:
   block0_consensus: genesis_praos
 
   # Number of slots in each epoch
-  slots_per_epoch: 3
+  slots_per_epoch: 20
```

Comment:
I was wondering how this will affect `STAKE_POOLS_LIST` tests... I've checked that it only makes tests run a bit longer (+10s)

Reply:
Yes, sorry I should have mentioned that in the PR comment (I also checked for this)